docs/modules/ws2812.md

```diff
@@ -44,6 +52,10 @@ ws2812.write({pin =  4, data = string.char(255, 0, 0, 255, 0, 0)},
              {pin = 14, data = string.char(0, 255, 0, 0, 255, 0)}) -- turn the two first RGB leds to green on the first strip and red on the second strip
 ```
 
+```lua
+ws2812.write({pin = 8, reset = 800, t0h = 3, t0l = 9, t1h = 6, t1l = 6, data = string.char(1, 0, 0)}) -- turn the SK6812 RGB led on the ESP32-C3-DevKitM-1 to green
```

Comment:
Does this work -- I would have thought that using 255 as the intensity rather than 1. Also, what is the color ordering for the SK6812? You set the first value to get green, which implies that it isn't RGB.

Reply:
Yeah -- that makes sense